owners/index.js

```diff
@@ -112,7 +112,15 @@ module.exports = app => {
   });
 
   /** Probot request handlers **/
-  app.on(['pull_request.opened', 'pull_request.synchronize'], async context => {
+  app.on(['pull_request.opened'], async context => {
+    await ownersBot.runOwnersCheck(
+      GitHub.fromContext(context),
+      PullRequest.fromGitHubResponse(context.payload.pull_request),
+      /* requestOwners=*/ true
```

Comment:
nit: I think the common style here is to write `/* requestOwners */` without the equals sign. Not 100% sure if we have a style-guide rule for this